src/styles/partials/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__nav__item {
+    font-weight: 500;
+  }
+
+  .nav__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  .nav__link {
```

Comment:
Add font-weight here instead of nav__item, maybe this will fix the tests

Reply:
All font-related styles are in this class so why would font-weight be in nav__item class?